app/src/main/java/it/quip/android/actvitiy/CreateQuipActivity.java

```diff
@@ -0,0 +1,104 @@
+package it.quip.android.actvitiy;
+
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import java.util.List;
+
+import it.quip.android.QuipitApplication;
+import it.quip.android.R;
+import it.quip.android.fragment.CreateQuipCircleSelectFragment;
+import it.quip.android.fragment.CreateQuipComposeFragment;
+import it.quip.android.model.Circle;
+import it.quip.android.model.Quip;
+import it.quip.android.model.User;
+import it.quip.android.util.TimeUtils;
+
+public class CreateQuipActivity
+        extends AppCompatActivity
+        implements CreateQuipComposeFragment.OnComposeQuipListener {
+
+    private static final String COMPOSE = "compose_fragment";
+
+    private CreateQuipComposeFragment mCreateQuipComposeFragment;
+    private CreateQuipCircleSelectFragment mCreateQuipCircleSelectFragment;
```

Comment:
Same thing here. If we ever want to use this outside of the creation flow (eg. editing a quip and sharing with more circles) it might be better if this was just `QuipCircleSelectFragment`